resources/js/components/CustomersTable.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+    <loading-view :loading="initialLoading">
+        <loading-card :loading="loading" class="card relative">
+            <table
+                v-if="customers.length > 0"
+                class="table w-full"
+                cellpadding="0"
+                cellspacing="0"
+                data-testid="resource-table"
+            >
+                <thead>
+                    <tr>
+                        <th class="text-left">
+                            <span class="inline-flex items-center">
+                                Customer ID
+                            </span>
+                        </th>
+                        <th class="text-left">
+                            <span class="inline-flex items-center"> Name </span>
+                        </th>
+                        <th class="text-left">
+                            <span class="inline-flex items-center">
+                                Email
+                            </span>
+                        </th>
+                        <th class="text-left">
+                            <span class="inline-flex items-center">
+                                Balance
+                            </span>
+                        </th>
+                    </tr>
+                </thead>
+
+                <tbody v-for="customer in customers">
+                    <tr>
+                        <td>{{ customer.id }}</td>
+                        <td>{{ customer.name }}</td>
+                        <td>{{ customer.email }}</td>
+                        <td>
+                            <span v-if="customer.currency">
+                                {{ customer.currency | money(customer.balance) }}
+                            </span>
+                            <span v-else>-</span>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
```

Comment:
We might want to add an empty state that just says something like "No customers"